### ISSUE TITLE
Create a new attestation github repository where 3rd party can contribute a CDXA attestation document as a PR

### ISSUE THREAD

**Issue by andrew-m-leonard** [adoptium/temurin-build#4054]
A new adoptium repository is required for the new CDXA process: https://github.com/adoptium/temurin-build/issues/3949#issuecomment-2507453653

**Repository name:** **github.com/adoptium/temurin-attestations**

**Layout:**
- <major_version>/jdk_\<XX_0_YY_BB\>\_\<platform\>\_\<3rd-party-org-name\>.xml - CDXA document
- <major_version>/jdk_\<XX_0_YY_BB\>\_\<platform\>\_\<3rd-party-org-name\>.xml.sign.pub - Public signing key
- <platform> can be one of the supported and reproducible platforms and versions (jdk-21+), and the naming uses the existing https://github.com/adoptium/temurin21-binaries/releases asset platform naming
      - x64_linux
      - aarch64_linux
      - ppc64le_linux
      - s390x_linux
      - x64_windows
      - x64_mac
 - The xml is signed using a standard "Enveloped XML Signature" of the \<bom\>, using such tools as:
   - cyclonedx-cli : https://github.com/CycloneDX/cyclonedx-cli
   - xmlxsec1 : https://linux.die.net/man/1/xmlsec1
   -  

eg:
```
21/jdk_21_0_5_11_x64_linux_AcmeInc.xml
21/jdk_21_0_5_11_x64_linux_AcmeInc.xml.sign.pub
21/jdk_21_0_5_11_x64_linux_OtherOrgLtd.xml
21/jdk_21_0_5_11_x64_linux_OtherOrgLtd.xml.sign.pub
```

**Example CDXA.xml** : https://github.com/adoptium/temurin-build/issues/3950#issuecomment-2503602280

**Pull Request github action will run that:**
- Validates the CDXA.xml format using cyclonedx-cli validate
- Verifies the Signature of the .xml using the supplied organization public key .sign.pub
- Validates the CDXA bom content:
  - Contains a "claim" for "VERIFIED_REPRODUCIBLE_BUILD"
  - "claim" references a "target component" with a valid target:
    - Valid "name" (Temurin JDK_BUILD PLATFORM) 
    - Valid "version"
    - Valid "platform"
    - Verifies CDXA "target component" "url" matches "https://api.adoptium.net/v3/binary/version/\<release\>/\<os\>/\<arch\>/jdk/hotspot/normal/eclipse"
    - Verifies CDXA "target component" "hash" matches the adoptium checksum "https://api.adoptium.net/v3/checksum/version/\<release\>/\<os\>/\<arch\>/jdk/hotspot/normal/eclipse"
  - Contains an "assessor" with an organization name
  - Contains an "attestation" by "assessor" for "claim"
  - Contains an "affirmation" with a "statement"
- Validate xml filename and path are valid version and architecture, and match the CDXA "target component" "version" and "platform"
-

**Comment by andrew-m-leonard** [comment]
@jiekang @smlambert @tellison For comment please : https://github.com/adoptium/temurin-build/issues/4054#issue-2683053258

**Comment by jiekang** [comment]
Is there an example CXDA attestation that can be looked at in reference to this?

**Comment by jiekang** [comment]
I would lean towards a shorter repo name: github.com/adoptium/temurin-attestations

**Comment by andrew-m-leonard** [comment]
> Is there an example CXDA attestation that can be looked at in reference to this?

@jiekang yes here: https://github.com/adoptium/temurin-build/issues/3950#issuecomment-2503602280

**Comment by jiekang** [comment]
Thanks. Looks fine to me!

**Comment by andrew-m-leonard** [comment]
New repository otterdog request: https://github.com/adoptium/.eclipsefdn/pull/74

**Comment by andrew-m-leonard** [comment]
Complete

**Comment by andrew-m-leonard** [comment]
From implementing the api.adoptium.net changes to [support Attestations](https://github.com/adoptium/api.adoptium.net/pull/1370/files), i've realized the folder organization of the CDXA documents:
```
<major_version>/jdk_<XX_0_YY_BB>_<platform>_<3rd-party-org-name>.xml - CDXA document
```
eg:
```
21/jdk_21_0_5_11_x64_linux_Adoptium.xml
```
Is a potential future problem with github GraphQL file querying, as the content of a LTS major version could potentially have a large number of files. With not being able to "page" a directory file query it would become a problem, i've estimated that github GraphQL file query would fail if it tried to list a folder containing 1000+ files, mainly due to response size.

I am proposing changing the temurin-attestations document folder structure to the following:
```
<major_version>/<release name>/jdk_<XX_0_YY_BB>_<platform>_<3rd-party-org-name>.xml - CDXA document
```
eg.
```
21/jdk-21.0.5+11/jdk_21_0_5_11_x64_linux_Adoptium.xml
```

I think this would actually be a better organization as well, especially with the potential for growth in "Verifiers" and automation of verifications by vendors.
@smlambert @tellison fyi
